bundles/org.openhab.binding.smsmodem/README.md

```diff
@@ -0,0 +1,129 @@
+# SMSModem Binding
+
+This binding connects to a USB serial GSM modem (or a network exposed one, see ser2net) and allows openHAB to send and receive SMS through it.
+
+Serial modem should all use the same communication protocol (AT message) and therefore this binding _should_ be compatible with every dongle. However, there is a gap between theory and reality and success may vary. The protocol stack is based on the no longer supported smslib project (more precisely a v4 fork), and all modems supported by this library should be OK. The following devices have been reported functional :
+
+-  Huawei E180
+
+## Supported Things
+
+Two things are supported by this binding :
+
+- A *smsmodembridge*, representing the dongle
+- A *smsconversation*, representing a conversation between one distant msisdn and the msisdn on the sim card in the dongle.
+
+## Discovery
+
+There is no discovery process for *smsmodembridge* thing.
+A *smsconversation* thing will be discovered and added to the inbox everytime the modem should receive a SMS by a new sender.
+
+## Thing Configuration
+
+The *smsmodembridge* thing requires at least two parameters to work properly (serialPortOrIP, baudOrNetworkPort).
+Depending on the nature of the connection (direct serial modem, or serial over network), this two fields will be used differently :
+
+| field | direct serial modem   | serial over network                  |
+|-------|--------------------------|-----------------------------------|
+|serialPortOrIP| The serial port to access (eg. /dev/tty/USBx) | IP address of the computer hosting the ser2net service|
+|baudOrNetworkPort| Baud rate                                 | The network port of the ser2net service |
+
+The other parameters are optional :
+
+| field | description     |
+|-------|--------------------------------------|
+|simPin          |If your sim card is protected, fill this field with the PIN code|
+|pollingInterval| Delay between two checks for new message|
+|delayBetweenSend|Delay to wait between two messages post (could be necessary for slow modem)|
```

Comment:
You could specify the type of the parameter (String, Decimal, ...) and the unit (seconds, ...).

Reply:
Indeed ! Done